cluster-autoscaler/processors/datadog/common/common.go

```diff
@@ -61,7 +68,22 @@ func SetNodeLocalDataResource(nodeInfo *schedulerframework.NodeInfo) {
 	if node.Status.Capacity == nil {
 		node.Status.Capacity = apiv1.ResourceList{}
 	}
-	node.Status.Capacity[DatadogLocalDataResource] = DatadogLocalDataQuantity.DeepCopy()
-	node.Status.Allocatable[DatadogLocalDataResource] = DatadogLocalDataQuantity.DeepCopy()
+
+	provisioner, _ := node.Labels[DatadogLocalStorageProvisionerLabel]
+	switch provisioner {
+	case "topolvm":
+		capacity, _ := node.Labels[DatadogInitialStorageCapacityLabel]
+		capacityResource, err := resource.ParseQuantity(capacity)
+		if err == nil {
+			node.Status.Capacity[DatadogLocalDataResource] = capacityResource.DeepCopy()
+			node.Status.Allocatable[DatadogLocalDataResource] = capacityResource.DeepCopy()
+		} else {
+			klog.Warningf("failed to attach storage information to node (%s): %v", node.Name, err)
```

Comment:
Can we add the bogus/unparsable capacity string to that log line, to ease diagnostics?